lib/stats.js

```diff
@@ -532,3 +438,42 @@ var stopProcessUsageReporting = function() {
 // Export startProcessUsageReporting and stopProcessUsageReporting
 exports.startProcessUsageReporting  = startProcessUsageReporting;
 exports.stopProcessUsageReporting   = stopProcessUsageReporting;
+
+
+/**
+ * Create a stats handler for taskcluster-client clients which takes an option
+ * `stats` as function that will be call after an API call.
+ *
+ * options:
+ * {
+ *   tags: {                         // Tags as key/value (both strings)
+ *     component: 'queue',           // Component identifier
+ *     process:   'web'              // Process identifier
+ *     // Common tags that are good to use includes:
+ *     //     component, process, provisionerId, workerType
+ *   },
+ *   drain:        new Influx(...),  // Place to send events
+ * }
+ */
+var createAPIClientStatsHandler = function(options) {
+  options = _.defaults(options || {}, {
```

Comment:
Is there a difference between `_.defaults(options || {}, { ...})` or `_.defaults({}, options, {...})` ? Just curious